tests/testTypescript.tsx

```diff
@@ -30,4 +30,4 @@ const TestFlow = () => (
   </Value>
 );
 
-() => <TestFlow />;
+export const Component = () => <TestFlow />;
```

Comment:
Will `export const TestFlow` work too?

Reply:
Yep, it complains about missing display name